README.md

```diff
@@ -1,53 +1,45 @@
-`odo` - Developer-focused CLI for Kubernetes and OpenShift
+`odo` - Fast iterative application development deployed immediately to your Kubernetes cluster
 ---
 
 [![GitHub release](https://img.shields.io/github/v/release/redhat-developer/odo?style=for-the-badge)](https://github.com/redhat-developer/odo/releases/latest)
 ![License](https://img.shields.io/github/license/redhat-developer/odo?style=for-the-badge)
 [![Godoc](https://img.shields.io/badge/godoc-reference-007d9c?logo=go&logoColor=white&style=for-the-badge)](https://odo.dev/godoc)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/e07867b0-56a4-4905-92a9-a152ceab5f0d/deploy-status)](https://app.netlify.com/sites/odo-docusaurus-preview/deploys)
 
+![logo](/docs/website/static/img/logo_small.png)
+
+----
 
 ### Overview
 
 `odo`  is a fast, iterative, and straightforward CLI tool for developers who write, build, and deploy applications on Kubernetes and OpenShift.
 
-Existing tools such as `kubectl` and `oc` are more operations-focused and require a deep-understanding of Kubernetes and OpenShift concepts. `odo` abstracts away complex Kubernetes and OpenShift concepts for the developer.
-
-### Key features
-
-`odo` is designed to be simple and concise with the following key features:
+Why use `odo`?
 
-* Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
-* Completely client based. No additional server other than Kubernetes or OpenShift is required for deployment.
-* Official support for Node.js and Java components.
-* Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the cluster.
+* **Fast:** Spend less time maintaining your deployment infrastructure and more time coding. Immediately have your application running each time you compile.
```

Comment:
> Spend less time maintaining your deployment infrastructure

This is not accurate. Isn't "deployment infra" generally the thing that Ops maintain in prod? 

> each time you compile

Each time you save, no? odo builds the binary using the `build` command, no?